extendr-api/src/wrapper/mod.rs

```diff
@@ -238,6 +238,11 @@ make_getsexp!(Dataframe<T>, impl<T>);
 //     }
 // }
 
+pub trait Elt<T> {
+    fn elt(&self, index: usize) -> T;
```

Comment:
typically I see these methods more or less like `get()` and `get_unchecked()` do we want to support this where `elt()` should return `Result<T>` and `elt_unchecked()` would return `T`